engine-tests/src/main/java/org/terasology/unittest/stubs/ListOfEnumsComponent.java

```diff
@@ -2,12 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.terasology.unittest.stubs;
 
-import org.terasology.engine.entitySystem.Component;
 import org.terasology.engine.math.Side;
+import org.terasology.gestalt.entitysystem.component.Component;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class ListOfEnumsComponent implements Component {
+public class ListOfEnumsComponent implements Component<ListOfEnumsComponent> {
     public List<Side> elements = new ArrayList<>();
+
+    @Override
+    public void copy(ListOfEnumsComponent other) {
+        this.elements = other.elements;
```

Comment:
Is this a "deep copy"? From the [gesalt wiki](https://github.com/MovingBlocks/gestalt/wiki/Gestalt-Entity-System-Quick-Start)

> Copies should be deep, so that components do not end up sharing references to mutable objects.